modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -36,6 +36,25 @@ zypper up susemanager
 +
 For more information about PostgreSQL database migration, see xref:db-migration-12.adoc[].
 
+. To migrate the database to PostgreSQL 13 run the migrate script:
++
+----
+/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh
+----
++
+For more information about PostgreSQL database migration, see xref:db-migration-13.adoc[].
+
+[NOTE]
+====
+Postgresql 12 or postgresql 13?
+
+This depends on the latest available version on the underlying operating system.
+Leap 15.2 has version 12 as the latest available version. 15.3 is expected to have postgresql 13.
```

Comment:
Hmm, my expectation is: with Leap 15.2 we will stay on postgresql 12 even if version 13 gets released there.
At the point in time when Uyuni switch to Leap 15.3 this DB migration will be required. But that will happen in about 6 month.
Not sure how to lower the user expectations here. We might need to check also with @paususe if we want to introduce pg13 in Uyuni earlier to get test results.

Reply:
I've rephrase it to state that we use pg12 with leap 15.2, instead of saying we use the latest version available.